DEVELOPER_CERTIFICATE_OF_ORIGIN.md

```diff
@@ -0,0 +1,38 @@
+This Developer Certificate of Origin (DCO) is a copy of version 1.1 of the DCO which originated from https://developercertificate.org/.
+
+```
+Developer Certificate of Origin
+Version 1.1
+
+Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
```

Comment:
Is this right?

Reply:
This is a copy of the DCO from https://developercertificate.org/, changing the document isn't authorized but that doesn't matter with how general the document is.
The copyright is for the document itself.